prowler/providers/aws/services/kafka/kafka_cluster_encryption_at_rest_uses_cmk/kafka_cluster_encryption_at_rest_uses_cmk.py

```diff
@@ -0,0 +1,31 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.kafka.kafka_client import kafka_client
+from prowler.providers.aws.services.kms.kms_client import kms_client
+
+
+class kafka_cluster_encryption_at_rest_uses_cmk(Check):
+    def execute(self):
+        findings = []
+
+        for arn_cluster, cluster in kafka_client.clusters.items():
+            report = Check_Report_AWS(self.metadata())
+            report.region = cluster.region
+            report.resource_id = cluster.id
+            report.resource_arn = arn_cluster
+            report.resource_tags = cluster.tags
+            report.status = "FAIL"
+            report.status_extended = f"Kafka cluster '{cluster.name}' does not have encryption at rest enabled with a CMK."
```

Comment:
Would it be interesting to put a FAIL if it is encrypted with another type of key?

Reply:
There is only two types of managers or AWS or CUSTOMER, if the manager is AWS it FAIL and if is CUSTOMER like is reflected on check it will PASS. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kms/client/describe_key.html